apps/api/migrations/074_add_vault_name_description_search.up.sql

```
@@ -0,0 +1,10 @@
+ALTER TABLE vaults ADD COLUMN IF NOT EXISTS name TEXT;
+ALTER TABLE vaults ADD COLUMN IF NOT EXISTS description TEXT;
+
+ALTER TABLE vaults ADD COLUMN IF NOT EXISTS search_vector tsvector
+    GENERATED ALWAYS AS (
+        setweight(to_tsvector('english', coalesce(name, '')), 'A') ||
+        setweight(to_tsvector('english', coalesce(description, '')), 'B')
+    ) STORED;
+
+CREATE INDEX IF NOT EXISTS idx_vaults_search_vector ON vaults USING GIN (search_vector);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)